pkg/output/stdout.go

```diff
@@ -155,13 +155,28 @@ func (s *Stdoutput) resultQuiet(resp ffuf.Response) {
 	}
 }
 
+func (s *Stdoutput) getRedirectLocation(resp *ffuf.Response) string {
+
+	redirectLocation := ""
+	if resp.StatusCode >= 300 && resp.StatusCode <= 399 && s.config.ShowRedirectLocation == true {
+		redirectLocation = resp.Headers["Location"][0]
+	}
+
+	return redirectLocation
+}
+
 func (s *Stdoutput) resultNormal(resp ffuf.Response) {
 	var res_str string
+
+	redirectLocation := s.getRedirectLocation(&resp)
+	if redirectLocation != "" {
+		redirectLocation = fmt.Sprintf(", Redirect: %s", redirectLocation)
+	}
```

Comment:
I would prefer seeing this logic wrapped into a function alike `colorizeStatus` as it would keep the conditional logic outside of the `resultNormal` function here.